### DISCUSSION TITLE
What's the mechanism behind the current modeling of food effect?

### DISCUSSION THREAD

**Discussion by regulus92** [Open-Systems-Pharmacology/Forum#1051] [Questions & Problems]
Hi, everyone. I'd like to ask for some advice on food effect.  For example, we already know that food intake does not affect absorption and bioavailability of clarithromycin. But when using clarithromycin model from the OSP templates, simultaneous food intake significantly reduce its plasma concentration (under multiple doses). This introduces inaccuracy in some simulation scenarios such as DDI, if a victim drug should be administered with food.

**Comment by StephanSchaller** [comment]
Can you show a graph of the model mismatch to data?

Absorption may be delayed due to an increased gastric emptying time. This does not necessarily reduce AUC but possibly Cmax and extend Tmax.
What are you expecting and how is the model not delivering?

Thanks for clarifying,
Stephan

**Reply by regulus92** [reply]
@StephanSchaller 
![1638949516(1)](https://user-images.githubusercontent.com/45459593/145168787-cd8f3789-6a50-4497-a152-1f6361771b0f.png)

**Reply by StephanSchaller** [reply]
Did you compare AUC of a single dose w/wo food?

As mentioned, I am pretty sure if food affects gastric emptying, it will affect Cmax of any drug.

**Reply by StephanSchaller** [reply]
Also, have you checked the publication? Are there any validations for clarithromycin with/without food? How does the model compare?
Best,
Stephan

**Reply by regulus92** [reply]
Thank you for your reply. AUC of of a single dose has only a slight increase, while Tmax is extended.
When I set coadministration with food in 'events' without revising other parameters like gastric emptying time, will these parameters be automatically changed in the simulation?

**Comment by regulus92** [comment]
Besides, I don't figure out why food intake has an effect even 20 hours later. Please see the example of efavirenz (also from template) below.
![1638947356(1)](https://user-images.githubusercontent.com/45459593/145164525-25458b59-59cc-4965-a3d4-9d03702af828.png)

**Reply by StephanSchaller** [reply]
Yes, this is a known issue (not sure if there exists an issue entry) for some meals where the function describing the gastric emtying time (GET) after food intake does not return to the fasted GET value also long after the meal event and thus the GET is reset to fasted value after some time.

**Comment by regulus92** [comment]
Most template inducers/inhibitors are administered under the fasted state. We are attempting to provide DDI predictions for a new drug candidate. While this drug should be co-ministered with food, I'm worried that current template models may have predictive bias with food effects.
